### ISSUE TITLE
Training ImageReward model on different budgets

### ISSUE THREAD

**Issue by bhattg** [THUDM/ImageReward#50]
Hi!
The paper mentions that the training for the ImageReward model is not easy and is sensitive to hyperparameters. In the section about hyperparameters, it says --" We find that fixing 70% of transformer layers with a learning rate of 1e-5 and batch size of 64 can reach up to the best preference accuracy."

Is this for the 8k budget? Can we get the suitable hyperparams for the other budgets? 

Secondly, which part of the code freezes the transformer layers? Thanks!

**Comment by xujz18** [comment]
Thanks for your discussion!
Firstly, the hyperparameters are for 8k budget (but the shuffle result may differ).
Secondly, see https://github.com/THUDM/ImageReward/blob/main/train/src/ImageReward.py#L87-L99.

**Comment by bhattg** [comment]
Thank you very much!

**Comment by bhattg** [comment]
Hey will it be possible to provide the hyperparams for 1k and 4k settings as well? That will be very useful.

**Comment by xujz18** [comment]
The 8k hyper-parameters should only need smaller adjustments to accommodate 1k/2k/4k.

**Comment by bhattg** [comment]
Thanks! In your experience which hyperparameters were the most sensitive? I will try to tune them.
